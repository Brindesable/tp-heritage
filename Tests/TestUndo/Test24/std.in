C circle1 0 0 10
C circle2 0 0 5
S selection -20 -20 20 20
MOVE selection 10 10
UNDO
LIST
