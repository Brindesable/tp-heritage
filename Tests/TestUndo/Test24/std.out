OK
#1 object(s) created
OK
#1 object(s) created
OK
#2 object(s) selected
OK
#2 object(s) moved
OK
#2 object(s) moved
C circle1 0 0 10
C circle2 0 0 5
